src/commands/dev/index.ts

```diff
@@ -56,6 +58,16 @@ export default async function main(ctx: NowContext) {
     return 2;
   }
 
+  const pkg = await readPackage();
+  if (pkg) {
+    const { scripts } = pkg as Package;
+
+    if (scripts && scripts.dev && /\bnow\b\W+\bdev\b/.test(scripts.dev)) {
```

Comment:
The regex testing logic should be a **NOT** or it always throws the following exception...